src/test/pybind/test_ceph_argparse.py

```diff
@@ -416,10 +416,10 @@ def test_incompat_rm_incompat(self):
 
 
 class TestFS(TestArgparse):
-    
```

Comment:
This and a bunch of other whitespace changes seem unrelated to the point of this patch.

Reply:
Hi @phlogistonjohn, many thanks for the feedback.

My editor is purposefully configured to clean dangling whitespace by default.
I suppose that my brain doesn't parse those has changes anymore.

I can revert all the whitespace cleanup if that is a real issue and cause for concern.

Reply:
Ideally, split them into two patches. One for the formatting change and one for the name change.